bus/bus.go

```diff
@@ -2339,15 +2311,50 @@ func (b *bus) multipartHandlerListPartsPOST(jc jape.Context) {
 	jc.Encode(resp)
 }
 
+// ExplicitCoveredFields returns a CoveredFields that covers all elements
+// present in txn.
+func ExplicitCoveredFields(txn types.Transaction) (cf types.CoveredFields) {
```

Comment:
This should go into `coreutils`. Right now even `hostd` is importing `renterd` to use this helper when it should be in the same place as the method we use it for.